packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -130,6 +130,7 @@ export class StatusBarImpl extends ReactWidget implements StatusBar {
             const rp = right.priority || 0;
             return rp - lp;
         });
+        elements.push(elements.splice(elements.findIndex(el => el.command === 'core.toggle.bottom.panel'), 1)[0]);
```

Comment:
@svenefftinge Would it be alright to sort like that or it is better to adjust priority for instance to negative max int?

Reply:
It should indeed be done in the declaration of the command, using a negative priority. Otherwise nobody could ever put something right to it.

Reply:
I.e. https://github.com/uniibu/theia/blob/9f33a31e9ccd30b80c8a8a8712f1ff7b9f47ac01/packages/core/src/browser/shell/application-shell.ts#L1098

Reply:
@svenefftinge is there a max negative priority? or is it `Number.MIN_SAFE_INTEGER` ? Simply putting `-1` to `core.toggle.bottom.panel`'s priority will not work as with the example with `prettier-vscode` it also uses `-1`, and this gets pushed last. Having said that, if using `Number.MIN_SAFE_INTEGER` or any other negative value, and a plugin also uses the same or much lower, the `core.toggle.bottom.panel` will no longer be the last(most right) item.

Reply:
I reverted previous commits and squashed. I decided to just follow what vscode does, which is to set the priority to `-1000` just like what it does with it's notification status bar item. https://github.com/Microsoft/vscode/blob/314cc521d8ce0c55773850ef520095afde33a62c/src/vs/workbench/browser/parts/notifications/notificationsStatus.ts#L79 

This still makes the toggle bottom panel to be rendered at most right 'most of the time', and still opens Theia to be customized by plugins/extensions rather than having toggle bottom panel being always at the right with no way of moving it.